examples/extended/create.sh

```diff
@@ -5,7 +5,8 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+#wait for appcontroller pod creation
+sleep 20
```

Comment:
For me on minikube with downloading fresh image from DockerHub this takes up to 2 minutes. Mb, we should be smarter or more flexible here. We could merge it now, but should think of it anyway.